pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -338,6 +338,16 @@ def test_setitem_bool_with_numeric_index(self, dtype):
 
         tm.assert_index_equal(df.columns, expected_cols)
 
+    @pytest.mark.parametrize("indexer", ["B", ["B"]])
```

Comment:
something off about the parameters here

Reply:
? I wanted to pass in ``"B"`` as string and as a list ``["B"]``, to check that result is equal after fixing the bug

Reply:
right you have an extra `[` i think then (on the string one), so maybe not doing what you expect

Reply:
I might misunderstand you here, but the indexer is ``"B"`` and ``["B"]`` when running the test, which is what I wanted to do

Reply:
I think @phofl is right here